src/main/java/me/svdev/mscategory/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,27 @@
+package me.svdev.mscategory.mapper;
+
+import me.svdev.mscategory.dao.entity.CategoryEntity;
+import me.svdev.mscategory.model.request.CategoryRequest;
+import me.svdev.mscategory.model.response.CategoryResponse;
+
+import static java.util.Optional.ofNullable;
+
+public enum CategoryMapper {
+    CATEGORY_MAPPER;
+    public static CategoryEntity mapRequestToEntity(CategoryRequest categoryRequest) {
+        return CategoryEntity.builder()
+                .name(categoryRequest.getName())
+                .isFavorite(categoryRequest.getIsFavorite())
+                .isVisible(true)
+                .build();
+    }
+
+    public static CategoryResponse mapEntityToResponse(CategoryEntity categoryEntity) {
+        return new CategoryResponse(categoryEntity.getId(), categoryEntity.getName(), categoryEntity.getIsFavorite(), null);
```

Comment:
bad practice to return null, use builder pattern or different classes for building response